recipes/r-multiblock/meta.yaml

```
@@ -0,0 +1,120 @@
+{% set version = '0.8.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-multiblock
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/multiblock_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/multiblock/multiblock_{{ version }}.tar.gz
+  sha256: 2273f0ae3ffee57b5af7b58e4f3123f1854a72cc0c82849b2329b8eb0dec29f6
+
+build:
+  skip: true  # [win]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem            # [win]
+    - {{ posix }}make
+    - {{ posix }}sed                   # [win]
+    - {{ posix }}coreutils             # [win]
+    - {{ posix }}zip                   # [win]
+    - cross-r-base {{ r_base }}        # [build_platform != target_platform]
+    - {{ cdt('mesa-libgl-devel') }}    # [linux]
+    - {{ cdt('mesa-dri-drivers') }}    # [linux]
+    - {{ cdt('libselinux') }}          # [linux]
+    - {{ cdt('libxdamage') }}          # [linux]
+    - {{ cdt('libxxf86vm') }}          # [linux]
+    - {{ cdt('libxext') }}             # [linux]
+  host:
+    - xorg-libxfixes                   # [linux]
+    - r-base
+    - r-mass
+    - r-rspectra
+    - r-rcpp
+    - r-rcppeigen
+    - r-ssbtools
+    - r-ade4
+    - r-car
+    - r-lme4
+    - r-mixlm
+    - r-plotrix
+    - r-pls
+    - r-plsvarsel
+    - r-pracma
+    - r-progress
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-mass
+    - r-rspectra
+    - r-rcpp
+    - r-rcppeigen
+    - r-ssbtools
+    - r-ade4
+    - r-car
+    - r-lme4
+    - r-mixlm
+    - r-plotrix
+    - r-pls
+    - r-plsvarsel
+    - r-pracma
+    - r-progress
+
+test:
+  commands:
+    - $R -e "library('multiblock')"           # [not win]
+    - "\"%R%\" -e \"library('multiblock')\""  # [win]
+
+about:
+  home: https://khliland.github.io/multiblock/, https://github.com/khliland/multiblock/
+  license: GPL-2.0-only
```

`# License: GPL (>= 2)` maps to
```suggestion
  license: GPL-2.0-or-later
```